auth/service.go

```diff
@@ -14,9 +14,6 @@ import (
 )
 
 const (
-	loginDuration    = 10 * time.Hour
-	recoveryDuration = 5 * time.Minute
```

Comment:
Don't remove recoveryDuration, please keep using the constant for recovery duration.

Reply:
It wasn't being used but I have kept it back